### DISCUSSION TITLE
Custom Theme in latest verion release 26.0.0

### DISCUSSION THREAD

**Discussion by simson-m** [keycloak/keycloak#33628] [Q&A]
I have been using custom theme in the keycloak. Three day ago new version of keycloak  26.0.0 has been released then after custom theme is not accept by the keycloak why. throwing error

ck: freemarker.core._TemplateModelException: An error has occurred when reading existing sub-variable "annotations"; see cause exception! The type of the containing value was: extended_hash+string (sun.reflect.generics.reflectiveObjects.TypeVariableImpl wrapped into f.e.b.StringModel)
----
FTL stack trace ("~" means nesting-related):
        - Failed at: #if !object[key]??  [in template "register.ftl" in function "ftl_object_to_js_code_declaring_an_object" at line 532, column 21]
        - Reached through: ${ftl_object_to_js_code_declaring_an_...  [in template "register.ftl" at line 6, column 17]
----

which goes on and makes keycloak image down why? 

any thing deprecated related ?

Help me sort it out, since the new version has many feature I would use it too.

**Comment by codespearhead** [comment]
Please close this issue [1] and reach out to the third-party Keycloak theme vendor you're using, because they're the ones who need to upgrade their product to the latest Keycloak version, not the other way around. We don't have control over their product.

Keycloak only supports its built-in themes.

[1] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
